test/test_vmap.py

```diff
@@ -596,6 +596,37 @@ def test_t(self):
         test(vmap(op), (torch.rand(B1, 2, B0, 5),), in_dims=2)
         test(vmap(vmap(op, in_dims=2)), (torch.rand(B1, 2, B0, 5, B2),), in_dims=2)
 
+    def test_no_random_op_support(self):
+        B0 = 2
+        random_ops = [
+            # out-of-place
+            (torch.bernoulli, (torch.rand(B0, 1),)),
```

Comment:
@mruberry Op DB!!!

Reply:
an op database would indeed solve a lot of my testing problems